langchain/src/callbacks/manager.ts

```diff
@@ -324,34 +359,36 @@ export class CallbackManager
   ): Promise<CallbackManagerForLLMRun> {
     let messageStrings: string[];
     await Promise.all(
-      this.handlers.map(async (handler) => {
-        if (!handler.ignoreLLM) {
-          try {
-            if (handler.handleChatModelStart)
-              await handler.handleChatModelStart?.(
-                llm,
-                messages,
-                runId,
-                this._parentRunId,
-                extraParams
-              );
-            else if (handler.handleLLMStart) {
-              messageStrings = messages.map((x) => getBufferString(x));
-              await handler.handleLLMStart?.(
-                llm,
-                messageStrings,
-                runId,
-                this._parentRunId,
-                extraParams
+      this.handlers.map((handler) =>
+        consumeCallback(async () => {
+          if (!handler.ignoreLLM) {
+            try {
+              if (handler.handleChatModelStart)
+                await handler.handleChatModelStart?.(
+                  llm,
+                  messages,
+                  runId,
+                  this._parentRunId,
+                  extraParams
+                );
+              else if (handler.handleLLMStart) {
```

Comment:
Does the linter not enforce putting brackets here? I think it's a bit hard to read

Reply:
I guess not, we need to review the linter config anyway, lots of silly rules in there